### PR TITLE
Use repository in project urls as a source

### PR DESCRIPTION
**Pull request by amazimbe**
### What are you trying to accomplish?
Dependabot opened a PR to bump the version of a python module, django-split-settings. However, in the body of the PR, the URL that has been created is wrong. It is a funding URL that has been configured in the package metadata. 

Customer is seeing a URL which is unrelated to the package. The URL is used to go to the package repository directly from the PR. However, with the wrong URL in place, the user is being redirected to the wrong place and getting a 404.

### How will you know you've accomplished your goal?
Dependabot will create the PR with the correct link.

### Checklist

- [x] I have run the complete test suite to ensure all tests and linters pass.
- [x] I have thoroughly tested my code changes to ensure they work as expected, including adding additional tests for new functionality.
- [x] I have written clear and descriptive commit messages.
- [x] I have provided a detailed description of the changes in the pull request, including the problem it addresses, how it fixes the problem, and any relevant details about the implementation.
- [x] I have ensured that the code is well-documented and easy to understand.

### CODE REVIEW

File: python/spec/fixtures/pypi/pypi_response_project_urls_source.json [hunk 1]
```diff
@@ -0,0 +1,712 @@
+{
```
Comment (amazimbe) [inline python/spec/fixtures/pypi/pypi_response_project_urls_source.json:1]: This is from the logs that reproduced the issue locally for me.

File: python/spec/dependabot/python/metadata_finder_spec.rb [hunk 2]
```diff
@@ -317,6 +317,12 @@
 
       it { is_expected.to eq("https://github.com/celery/celery") }
     end
+
+    context "when the dependency source is in project_urls" do
+      let(:pypi_response) { fixture("pypi", "pypi_response_project_urls_source.json") }
+
+      it { is_expected.to eq("https://github.com/bedford/django-split-settings") }
```
Comment (abdulapopoola) [inline python/spec/dependabot/python/metadata_finder_spec.rb:324]: this seems to be a private link too?

**Review by abdulapopoola — APPROVED** [review]
Approving provided the comments are resolved

File: python/spec/dependabot/python/metadata_finder_spec.rb (hunk 2 above)
Comment (amazimbe) [reply python/spec/dependabot/python/metadata_finder_spec.rb]: I used bedford as a placeholder but I've changed it to xxxxx to avoid confusion.

**Pull request merged**